drivers/xen/Kconfig

```diff
@@ -29,6 +29,12 @@ config XEN_GRANT_TABLE_INIT_PRIORITY
 	depends on XEN_GRANT_TABLE
 	default 50
 
+config XEN_EVENTS
```

Comment:
It will be good to take it to `zephyr-v3.6.0-xt` and upstream this change in separate PRs

Reply:
Will note is for integration. and Once we done with basic functionality - I'll revise commits and send those which are generic for integration.